thirdparty/aws-sdk-cpp/dll-export-injection.patch

```diff
@@ -0,0 +1,52 @@
+diff -rupN aws-sdk-cpp-src/aws-cpp-sdk-core/CMakeLists.txt aws-sdk-cpp-src-patched/aws-cpp-sdk-core/CMakeLists.txt
```

Comment:
I think this patch could use a comment for future reference.

Reply:
I feel like the name of the file is pretty talkative :)

Reply:
Yeah on second glance it should be explicit enough, it can be kept this way.